extra/crc32_armv8_neon/crc32_armv8.c

```diff
@@ -17,6 +18,13 @@ unsigned int crc32c_aarch64_available(void)
 	return (auxv & HWCAP_CRC32) != 0;
 }
 
+/* implementation is same just alias it to avoid confusion between crc32 and
+ * crc32c.*/
+unsigned int crc32_aarch64_available(void)
+{
+	return crc32c_aarch64_available();
+}
```

Comment:
Do we really need this? If you really think so, then please define it as an inline function or a macro.
Please indent code by 2 spaces instead of TAB.

Reply:
yes function is needed. added reasoning the code too. inlined as recommended.

Reply:
done. declaration now in my_sys.h